### ISSUE TITLE
Fail to fix - problem with comments and sniff TraitUseSpacing

### ISSUE THREAD

**Issue by ncou** [slevomat/coding-standard#1068]
Hi,

Nice piece of code !
I found a small bug in the sniff : SlevomatCodingStandard.Classes.TraitUseSpacing.IncorrectLinesCountBeforeFirstUse

If i add a comment using '//' or '#' on the first line in the class before the "use xxxTrait" instruction there is an error in the sniff it will report a negative lines count, and the fix command will fail. But if i use for the comments the notation '/*' or '/**' the sniff work fine.

Here is the error i got
`LINE 16: ERROR [x] Expected 0 lines before first use statement, found -1. (SlevomatCodingStandard.Classes.TraitUseSpacing.IncorrectLinesCountBeforeFirstUse)`

Here is a problem exemple.

```php
class Example
{
    // bad comment
    use MyCustomTrait;

   ....
}
```

Hope i am clear enought.
Keep up the good work.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/5fa352ad7adc3c0c77fa202dc6d595be098b9866
